Don't disconnect from certain roamable networks broadcasting multiple
SSIDs. See https://bugs.archlinux.org/task/61090 for more information.

diff -ur a/src/wl/sys/wl_cfg80211_hybrid.c b/src/wl/sys/wl_cfg80211_hybrid.c
--- a/src/wl/sys/wl_cfg80211_hybrid.c	2015-09-19 00:47:15.000000000 +0200
+++ b/src/wl/sys/wl_cfg80211_hybrid.c	2019-05-30 11:41:12.168748142 +0200
@@ -1446,7 +1446,6 @@
 
 	if (memcmp(mac, wl->profile->bssid, ETHER_ADDR_LEN)) {
 		WL_ERR(("Wrong Mac address, mac = %pM   profile =%pM\n", mac, wl->profile->bssid));
-		return -ENOENT;
 	}
 
 	err = wl_dev_ioctl(dev, WLC_GET_RATE, &rate, sizeof(rate));
